app/grandchallenge/challenges/templates/challenges/challenge_list.html

```diff
@@ -17,7 +18,7 @@
             </a>
         </div>
         <div class="col-xl-4 col-lg-5 col-md-6 mb-3 text-md-right">
-            {% if perms.challenges.change_challengerequest %}
+            {% if request.user|is_challenge_reviewer %}

```

Comment:
The object based permissions should be used here, see `get_perms_for_user` for other objects.

Reply:
In fact, you use it in the next file, why the difference?

Reply:
Oh! I see, again, add and use the global perm for this.

Reply:
Ok, I wasn't sure what would be better, checking for global permissions or checking for group membership. Why is it better to use global permissions here (apart from the fact that it makes the custom template tag unnecessary here)? And shouldn't we try and be consistent and use global permissions everywhere else then (also in the emails for example)?

Reply:
Yes it is better to use global permissions otherwise you need to duplicate things into the codebase about who has access, this is the shortcut for that so that we can set the authorisation in one place. Then it will also work if you want to give another group permission.

And yes, be consistent about it.